<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>two</class>
 <widget class="QDialog" name="two">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1010</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1010</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>图表显示</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/1/app.png</normaloff>:/1/app.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1011</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="fanhui">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>0</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background: transparent; 
	border: none; 
	color: rgb(211, 215, 207)

}</string>
    </property>
    <property name="text">
     <string>返回主界面</string>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/4/fanhui1.png</normaloff>:/4/fanhui1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="fanhui_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background: transparent; 
	border: none; 
	color: rgb(211, 215, 207)

}</string>
    </property>
    <property name="text">
     <string>开始绘制</string>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/1/huizhi.png</normaloff>:/1/huizhi.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="fanhui_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background: transparent; 
	border: none; 
	color: rgb(211, 215, 207)

}</string>
    </property>
    <property name="text">
     <string>暂停绘制</string>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/1/zantinghuizhi.png</normaloff>:/1/zantinghuizhi.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QCustomPlot" name="widget1" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>500</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
gridline-color: rgb(145, 101, 255);</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="widget2" native="true">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>50</y>
     <width>500</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">gridline-color: rgb(145, 101, 255);</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="widget3" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>500</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">gridline-color: rgb(145, 101, 255);</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="widget4" native="true">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>270</y>
     <width>500</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">gridline-color: rgb(145, 101, 255);</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
